examples/elemeno.yaml

```diff
@@ -0,0 +1,49 @@
+# This file contains the base configuration for the sdk to function properly
+# Elemeno-AI
+
+app:
+  mode: ${oc.env:ELEMENO_MODE,development}
+cos:
+  host: minio.app.elemeno.ai:9000
+  key_id: client_oneblinc
+  secret: m6bD8dUf$8vy
+  use_ssl: False
+  bucket: client-oneblinc
+registry:
+  tracking_url: http://mlflow.elemeno.ai
+feature_store:
+  feast_config_path: .
+  registry: s3://feature-view/client_oneblinc/feature-registry
+  sink:
+    type: Redshift
+    params:
+        cluster_name: "redshift-cluster-2"
+        host: "redshift-cluster-2.cwuptesab6o8.us-east-1.redshift.amazonaws.com"
+        port: 5439
+        user: "client_oneblinc"
+        password: "4v!Nh9Z4BS0G"
+        database: "client_oneblinc"
+  source:
+    type: Redshift
+    # params:
```

Comment:
Would you consider delete those statements, once they are commented?

Reply:
of course